sqlstore/src/main/resources/db/migration_beforeMigrate/05_standard.sql

```diff
@@ -419,6 +419,38 @@ CREATE PROCEDURE addTissueType(
   END IF;
 END//
 
+--  Use inconjunction with the spreadsheet formula: =CONCATENATE("CALL addIdentity('",A2,"', '",B2,"', '",C2,"', ", IF(ISBLANK(D2), "NULL", CONCATENATE("'", D2, "'")), ", '", E2, "', '", F2, "');")
+DROP PROCEDURE IF EXISTS addIdentity//
+CREATE PROCEDURE addIdentity(
+  iProject varchar(255),
+  iExternalName varchar(255),
+  iAlias varchar(255),
+  iDescription varchar(255),
+  iScientificName varchar(255),
+  iSampleType varchar(255)
+) BEGIN
+  DECLARE newSampleId bigint(20);
+  DECLARE errorMessage varchar(300);
+  
+  IF EXISTS (SELECT 1 FROM Identity JOIN Sample USING (sampleId) JOIN Project ON Project.projectId = Sample.project_projectId WHERE externalName = iExternalName AND Project.alias = iProject OR Sample.alias = iAlias) THEN
```

Comment:
`Project.alias` or `Project.shortName`? You use `Project.shortName` below, and we have 15 projects where alias != shortName. 

Might be worth renaming `iProject` to `iProject[Alias|ShortName]` to make that clearer for when we come back to this function in several weeks...